farm-ng-core.code-workspace

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
can this file be added  somehow only at c++ level ?

Reply:
yeah, I think I want to get rid of the whole super_project folder soon - similarly as I did for sophus2.